ios/Podfile.lock

```diff
@@ -23,6 +23,7 @@ PODS:
     - GoogleInterchangeUtilities (~> 1.2)
     - GoogleSymbolUtilities (~> 1.1)
     - GoogleToolboxForMac/Logger (~> 2.1)
+  - FLAnimatedImage (1.0.12)
```

Comment:
To insert splash animation for iOS app, I used framework.
https://github.com/Flipboard/FLAnimatedImage